src/Misc/tools.ts

```diff
@@ -17,6 +17,7 @@ import { InstantiationTools } from "./instantiationTools";
 import { GUID } from "./guid";
 import { IScreenshotSize } from "./interfaces/screenshotSize";
 import { SliceTools } from "./sliceTools";
+import { StringTools } from "./stringTools";

```

Comment:
This can be removed

Reply:
Indeed it can after the refactor. Removing it now.